include/CLI/Validators.hpp

```diff
@@ -212,11 +215,10 @@ class Validator {
             std::string s2 = f2(input);
             if(!s1.empty() && !s2.empty())
                 return std::string("(") + s1 + ") AND (" + s2 + ")";
-            else
-                return s1 + s2;
+            return s1 + s2;
         };
 
-        newval.active_ = (active_ & other.active_);
+        newval.active_ = ((static_cast<int>(active_) & static_cast<int>(other.active_)) != 0);
```

Comment:
this change seems odd?  maybe should be active_ && other.active_   instead of the int casts